src/components/TodoForm.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  loading: boolean,
+  createTodo: (newTodo: Todo) => void,
+  userId: number,
+  setNotification: (value: string) => void,
+  tempTodo: Todo | null,
+};
+
+export const TodoForm: React.FC<Props> = React.memo(
+  ({
+    loading,
+    createTodo,
+    userId,
+    setNotification,
+    tempTodo,
+  }) => {
+    const [title, setTitle] = useState<string>('');
+
+    const handelSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+
+      if (!title) {
```

Comment:
I have ability to add empty todo with spaces only
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/54936044/8e5ba15f-efab-4adf-b0b0-62ec83062e99)